crates/mero-devnet/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "mero-devnet"
```

Comment:
`calimero-sandbox`

Reply:
Is that what I should name the new crate?

Reply:
yes